crates/rslint_core/src/groups/errors/constructor_super.rs

```diff
@@ -0,0 +1,92 @@
+use crate::rule_prelude::*;
+use ast::{ClassDecl, ClassElement, Expr, Stmt};
+
+declare_lint! {
+    /**
+    Verify calls of `super()` in constructors (constructor-super)
```

Comment:
```suggestion
    Verify calls of `super()` in constructors
```
Adding the name is redundant since it's displayed in a table along with the name in the docs.